fix(dynamic permissions): fix incorrect doc on dynamic permissions deactivation

educhastenier opened the pull request.

educhastenier committed:
fix(dynamic permissions:) fix incorrect doc on dynamic permissions deactivation

abirembaut commented on modules/runtime/pages/bonita-docker-installation.adoc:
```diff
@@ -253,12 +253,12 @@ Now you can access the Bonita Runtime on localhost:8080/bonita and login using:
-This docker image ensures to activate by default both static and dynamic authorization checks on xref:identity:rest-api-authorization.adoc[REST API]. To be coherent it also deactivates the HTTP API.
-But for specific needs you can override this behavior by setting HTTP_API to true and BONITA_RUNTIME_AUTHORIZATION_DYNAMICCHECK_ENABLED to false :
+This docker image ensures to activate by default static (all editions) and dynamic authorization checks (Subscription editions only) on xref:identity:rest-api-authorization.adoc[REST API]. To be coherent it also deactivates the HTTP API.
+But for specific needs, like calling directly the Runtime APIs without calling the REST APIs, you can override this behavior by setting HTTP_API to true:
```
```suggestion
But for specific needs, like exposing directly the Runtime APIs through HTTP (not to be mixed up with the REST API), you can override this behavior by setting HTTP_API to true:
```

abirembaut reviewed (commented):
just a small improvement

educhastenier committed:
fix(links): fix invalid internal references

on existing page.
This prevents the PR to pass, even though I did not touch those links

educhastenier committed:
apply wording suggestions

Co-authored-by: abirembaut <anthony.birembaut@bonitasoft.com>
